recipes/jinja2/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version="2.8" %}
+
+package:
+  name: jinja2
+  version: {{ version }}
+
+source:
+  fn: Jinja2-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/J/Jinja2/Jinja2-{{ version }}.tar.gz
+  md5: edb51693fe22c53cee5403775c71a99e
+
+build:
+  number:  0
+  script: python setup.py install --old-and-unmanageable
```

Comment:
I don't reccall where @minrk and @pelson started discussing this, but they have

`python -m pip install --no-deps .`

in some of the Jupyter/IPython feedstocks. (Then they substitute `setuptools` for `pip` in the build dependency as setup is not needed when installing that way. Maybe even specifying `pip` is not needed as that comes by default with `conda`.)